source_code/src/AES/aes.c

```diff
@@ -22,10 +22,13 @@
 #define F(x)   (((x)<<1) ^ ((((x)>>7) & 1) * 0x1b))
 #define FD(x)  (((x) >> 1) ^ (((x) & 1) ? 0x8d : 0))
 
-#define BACK_TO_TABLES
 #ifdef BACK_TO_TABLES
 
-const uint8_t sbox[256] __attribute__ ((__progmem__)) = {		// forward s-box
+#if defined(__AVR__)
+const uint8_t sbox[256] __attribute__ ((__progmem__)) = {        // forward s-box
+#else
+const uint8_t sbox[256] = {        // forward s-box

```

Comment:
Yes. But this is for non avr devices. Aka a PC. This was required to use the same AES with my Host App. It doesnt hurt to make the AES also available on non AVR devices (without progmem) ;)